src/test/java/org/apache/commons/dbcp2/managed/TestManagedDataSource.java

```diff
@@ -198,9 +198,8 @@ public void testManagedConnectionEqualsSameDelegate() throws Exception {
     @Test
     public void testManagedConnectionEqualsReflexive() throws Exception {
         final Connection con = ds.getConnection();
-        final Connection con2 = con;
```

Comment:
Don't change there I think. See the title of the method. This makes it obvious.

Reply:
Done